covid_data.py

```diff
@@ -16,10 +16,8 @@
     get_tweets_from, pairwise, parse_file, parse_numbers, pptx2chartdata, seperate, split, toint, web_files, \
     web_links, all_in, NUM_OR_DASH
 from utils_thai import DISTRICT_RANGE, PROVINCES, area_crosstab, file2date, find_date_range, \
-    find_thai_date, get_province, join_provinces, parse_gender, to_switching_date, today, RAW_COLS, COLUMNS
-
-
-prov_guesses = pd.DataFrame(columns=["Province", "ProvinceEn", "count"])
```

Comment:
I don't agree with this move. The choose to collect the guesses and export them is really made by the callers of utils_thai not utils_thai itself so it doesn't make sense to have that global there.
There is an argument for not making it global however. It's not a constant.

Reply:
or it should be taken out of the arguments to get_province and join_province  and determined its just a sideeffect of using utils_thai that it will accumulate a list of guessed names as its used.